src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/pages/AddPricePage.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages
+
+import org.fluentlenium.core.annotation.PageUrl
+import org.fluentlenium.core.domain.FluentWebElement
+import org.openqa.selenium.support.FindBy
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Money
+
+@PageUrl("http://localhost:8080/add-price/{fromAgencyId-toAgencyId}")
+class AddPricePage : ApplicationPage() {
+
+  @FindBy(id = "price")
+  private lateinit var price: FluentWebElement
+
+  @FindBy(id = "confirm-save-price")
+  private lateinit var submit: FluentWebElement
+
+  fun isAtPricePageForJourney(fromAgencyId: String, toAgencyId: String) {
+    this.isAt("$fromAgencyId-$toAgencyId")
+  }
+
+  fun addPriceForJourney(fromAgencyId: String, toAgencyId: String, amount: Money) {
+    this.isAtPricePageForJourney(fromAgencyId, toAgencyId)
+    price.fill().withText(amount.toString())
+    submit.click()
```

Comment:
What does success look like here? In the tests I did not see us check for a confirmation page or something - what would happen if this errorred out, would the tests still pass?

Reply:
Yes we can...same goes for update.

![image](https://user-images.githubusercontent.com/60104344/124437894-72ad4b00-dd6f-11eb-96b7-398d07053ccf.png)